### PR TITLE
Ignored

### PR DESCRIPTION
**Pull request by jamestautges**
**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Ignored module paths are stored absolutely, causing portability issues (issue #2991).

**What is the new behavior?**
Relativise path before storing in records.

**Does this PR introduce a breaking change?**
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the following... 
- Impact:
- Migration path for existing applications: 
- Github Issue(s) this is regarding:

**Other information**:
I couldn't figure out how to make a setup log an ignored module, so I wasn't able to write a test. Anyone can help me out with reproducing the behavior, I'll happily update this with a test.

### CODE REVIEW

**Comment by jamestautges** [comment]
@TheLarkInn I'd love to write a test, but I don't know enough about this part of webpack to reproduce the issue. If you or someone else could tell me when modules get ignored, then I'll write the test.

File: lib/NormalModuleFactory.js [hunk 1]
```diff
@@ -121,7 +122,7 @@ function NormalModuleFactory(context, resolvers, options) {
 				if(resource === false)
 					return callback(null,
 						new RawModule("/* (ignored) */",
-							"ignored " + context + " " + request,
+							      "ignored " + path.resolve(__dirname, context) + " " + request,
```
Comment (sokra) [inline lib/NormalModuleFactory.js]: relative instead of resolve.

File: lib/NormalModuleFactory.js [hunk 2]
```diff
@@ -121,7 +122,7 @@ function NormalModuleFactory(context, resolvers, options) {
 				if(resource === false)
 					return callback(null,
 						new RawModule("/* (ignored) */",
-							"ignored " + context + " " + request,
+							      "ignored " + path.relative(__dirname, context) + " " + request,
```
Comment (sokra) [inline lib/NormalModuleFactory.js]: I wonder if `"ignored"` is enough here:

`new RawModule("/* (ignored) */", "ignored", "ignored")`

**Comment by sokra** [comment]
Take a look at `test/README.md` for information about writing tests

**Comment by sokra** [comment]
fixed by #4500

**Pull request closed without merging**